pkg/pilot/genericpilot/probe/listen.go

```diff
@@ -16,8 +18,12 @@ func (l *Listener) Listen() error {
 		fmt.Sprintf(":%d", l.Port),
 		http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			if err := l.Check(); err != nil {
+				glog.Errorf("Error '%s', returned by Check function for probe on port %d", err, l.Port)
```

Comment:
Can you re-arrange this so `err` is at the end? `failed running Check function for probe on port %d: %v` (also no quotation marks around the error message)